src/controllers/krate/search.rs

```diff
@@ -568,20 +575,34 @@ mod seek {
             &self,
             record: &(Crate, bool, i64, Option<i64>, f32),
         ) -> SeekPayload {
+            let (
+                Crate {
+                    id,
+                    updated_at,
+                    created_at,
+                    ..
+                },
+                exact_match,
+                downloads,
+                recent_downloads,
+                rank,
+            ) = *record;
```

Comment:
Some day I hope to be brave enough to write a `let` destructure like this. :laughing: